AUTHORS

```diff
@@ -4,3 +4,4 @@ James Moriarty <jamespaulmoriarty@gmail.com>
 Julia Ogris <julia.ogris@gmail.com>
 Keith Ferguson <keith.ferguson@anz.com>
 Sam Uong <samuong@gmail.com>
+Seng Ern Gan <sengern.gan@anz.com>
```

Comment:
You're missing an end of line at end of file (see the red marker in the github review page). Please add. https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b